src/common/i18n.js

```diff
@@ -44,9 +44,11 @@ export const translate = (input, params = []) => {
         const stringToBeTranslated = input.replace(/\{\$({0-9])\}/gi, '%$1');
         let translatedString = i18nTranslate(stringToBeTranslated);
         params.forEach((replacement, index) => {
-            translatedString = translatedString.replaceAll(`\{\$${index}\}`, replacement);
+            if (translatedString && typeof translatedString === 'string') {
+                translatedString = translatedString.replaceAll(`\{\$${index}\}`, replacement);
```

Comment:
is there any situation that the `translatedString` is not `string`
if so, do we need to handle this situation?

Reply:
Yes, I think sometime somehow the translateString in not string and something else.
Its been caught in the trackjs that r.replaceAll is not a function.
[trackjs](https://my.trackjs.com/errors/filtered?hash=d79fa8f299d70d61)
[image](https://user-images.githubusercontent.com/90243468/148722496-98fa0c29-915c-427e-92f1-017722c35966.png)


So if translateString is null or undefined then it should result `cannot read property replaceAll of null or undefined`.
instead its giving `r.replaceAll is not a function`.

I was unable to reproduce this trackjs error but I thought adding a type check before replaceAll would be a good idea.

Let me know your opinion about this.